### PR TITLE
Add initial pester tests

### PR DESCRIPTION
**Pull request by mattstratton**
This basically just adds some initial pester tests to cover the default recipe, and I started the skeleton of a fixture cookbook for testing the lwrp's.

I also made a change to the kitchen file so that it uses Server instead of Windows 8.1. And points to @jdmundrawala branch for kitchen-vagrant.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+source 'https://rubygems.org'
 
 group :development do
-  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
-  gem 'kitchen-vagrant', git: 'https://github.com/afiune/kitchen-vagrant.git', :branch => 'Transport'
-  gem "berkshelf"
-  gem "vagrant-wrapper", ">= 2.0"
-  gem "chefspec"
+  gem 'test-kitchen', :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
+  gem 'kitchen-vagrant', git: 'https://github.com/jdmundrawala/kitchen-vagrant.git', :branch => 'Transport'
```
Comment (jdmundrawala) [inline Gemfile]: I don't think you should be pointing at my branch... it should probably be Salim's

File: .kitchen.yml [hunk 2]
```diff
@@ -14,24 +14,34 @@ provisioner:
   name: chef_zero
 
 platforms:
-  - name: windows-8.1-chef11.10.4
+  # - name: windows_2008r2-chef11
```
Comment (jdmundrawala) [inline .kitchen.yml]: Any reason to keep this commented out code around?

File: Gemfile (hunk 1 above)
Comment (mattstratton) [inline Gemfile]: What do we have to do to get this into the test-kitchen one so we don't have this keep happening? I keep getting confused as to what is canonical these days.

File: .kitchen.yml (hunk 2 above)
Comment (mattstratton) [inline .kitchen.yml]: To make it possible to add it back in again once we have a vagrantbox that works again. But I'm cool with whacking it. I'll do so now.

File: Gemfile (hunk 1 above)
Comment (mattstratton) [inline Gemfile]: That said, I'm fine with switching to Salim's. I'll change it and re-run my tests just to be sure.

Comment (jdmundrawala) [inline Gemfile]: Ya, let's keep it consistent in opscode-cookbooks

I say change it then :shipit:

**Comment by btm** [comment]
rebase and :ship:

**Comment by ksubrama** [comment]
Merged in e494d5d2e95674023ab665b175fae15bf10d01a7

**Pull request closed without merging**